src/main/java/hw_7/Circle.java

```diff
@@ -0,0 +1,15 @@
+package hw_7;
+
+public class Circle {
```

Comment:
Do we really need this class? What is the difference between Circle and Circle2 classes?

Reply:
yes, because one 'circle' is for hw from ppt and another for hw from worddoc